### PR TITLE
#23, Now the app has all the qml code able to be translated

### PR DESCRIPTION
**Pull request by diegowald**
The texts now are translated, and, in case translations is not found a TBT: is prepended to original text.

### CODE REVIEW

File: VoltAir/qml/ui/LoadingScreen.qml [hunk 1]
```diff
@@ -108,6 +108,6 @@ UiComponent {
         width: loading.width
         height: 0.25 * loading.height
 
-        textElement.text: "Loading"
+        textElement.text: TR.value("Loading")
```
Comment (jo-elimu) [inline VoltAir/qml/ui/LoadingScreen.qml]: @diegowald How does this work? Does the `TR` utility look up a text stored in a text file with the property value "Loading"?

**Review by jo-elimu — CHANGES_REQUESTED** [review]

File: VoltAir/qml/ui/LoadingScreen.qml (hunk 1 above)
Comment (diegowald) [reply VoltAir/qml/ui/LoadingScreen.qml]: Hi Jo, 
yes, TR is a class that handles the translations. Internally has a dictionary, which, when invoked the value function, translates the enclosed text.
Currently, if not translated a TBT is preppended. (TBT: To be translated).

File: VoltAir/resources/values-Spanish.properties [hunk 2]
```diff
@@ -0,0 +1,16 @@
+Finish=Fin
```
Comment (jo-elimu) [inline VoltAir/resources/values-Spanish.properties]: @diegowald Change to lower-case property names, e.g. `finish=Fin`

File: VoltAir/resources/values-Spanish.properties [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+Connect controllers to play with your friends!=Conectar controles para jugar con tus amigos!
+No=No
+Yes=Yes
+Quit?=Salir?
+Loading=Cargando
+Resume=Continuar
+Main Menu=Menu Principal
```
Comment (jo-elimu) [inline VoltAir/resources/values-Spanish.properties]: @diegowald Don't use spaces in property names. Change to `main.menu=Menu principal`

File: VoltAir/resources/voltair.qrc [hunk 4]
```diff
@@ -0,0 +1,6 @@
+<RCC>
+    <qresource prefix="/"/>
+    <qresource prefix="/translations">
+        <file>values-Spanish.properties</file>
```
Comment (jo-elimu) [inline VoltAir/resources/voltair.qrc:4]: @diegowald Add English properties file.

**Review by jo-elimu — CHANGES_REQUESTED** [review]

File: VoltAir/resources/voltair.qrc (hunk 4 above)
Comment (diegowald) [reply VoltAir/resources/voltair.qrc:4]: Done!

Comment (diegowald) [reply VoltAir/resources/voltair.qrc:4]: Done!

File: VoltAir/resources/values-Swahili.properties [hunk 5]
```diff
@@ -0,0 +1,17 @@
+finish=Fin
```
Comment (jo-elimu) [inline VoltAir/resources/values-Swahili.properties]: @diegowald To make the file easier to translate, change its content from Spanish to English.

**Review by jo-elimu — CHANGES_REQUESTED** [review]

File: VoltAir/translations/tr.cpp [hunk 6]
```diff
@@ -52,6 +52,7 @@ QString TR::value(const QString &key)
     {
         loadDictionary();
     }
+    // TODO: Remove TBT when finishing translations.
     return _dictionary.contains(key) ? _dictionary[key] : QString("TBT: %1").arg(key);
```
Comment (jo-elimu) [inline VoltAir/translations/tr.cpp]: @diegowald Is this check still necessary now that English is the default fallback language?

**Pull request merged**